cubedash/testutils/database.py

```diff
@@ -100,7 +102,9 @@ def odc_db(postgresql_server, tmp_path_factory, request):
             tmp_path_factory.mktemp("odc") / "test_datacube.conf"
         )
         config = configparser.ConfigParser()
```

Comment:
With the new config API, it feels like there should be a simpler way to handle test config than dynamically generating a temporary ini file.

Reply:
Yeah, this whole setup is copypasta from odc-tools and needs to be revisited